libs/damap/src/lib/components/dashboard/dashboard.component.css

```diff
@@ -2,3 +2,12 @@
   font-size: 1.2em;
   padding: 1rem 1rem;
 }
+
+.content {
+  color: var(--primary-text);
+}
+
+.button-position-custom {
+  color: var(--white-smoke-text) !important;
+  font-weight: 400 !important;
+}
```

Comment:
Do we need to duplicate this class in the component if it is already specified in the global `styles.scss`? See also other components

Reply:
Have you tried it? Because is not being set otherwise.. il test it once more, otherwise i leave it here too.